packages/vx-demo/src/sandboxes/vx-xychart/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render } from 'react-dom';
+import ParentSize from '@vx/responsive/lib/components/ParentSize';
```

Comment:
Can this import from just `@vx/responsive`?

Reply:
yeah, but this is in the sandbox template / all of them are that way so would rather make a sweep to fix them all in the future.